_rules/iframe-accessible-name-cae760.md

```diff
@@ -30,7 +30,7 @@ Each target element has an [accessible name][] that is not empty (`""`).
 
 ## Assumptions
 
-- The rule assumes that the target `iframe` is used as an [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components).
+- The rule assumes that the target `iframe` is used as a [user interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components).
```

Comment:
I kinda disagree with that change 😕 
I assume that the an/a checker is thrown out by the square bracket.

Reply:
After double checking, "a user" is correct ("a you-zer", not "an uh-zer"). My bad.